### ISSUE TITLE
Merge into main build?

### ISSUE THREAD

**Issue by fallaha56** [Tinob/Ishiiruka#14]
Sorry if I'm missing something -amazing features like async shader compilation (big problem I understand for F-Zero and others) -why not merge the code into the main Dolphin builds?!

Thanks!

**Comment by Zetchzie** [comment]
Because the Main devs have ditched all this. They only want to focus on Modern machines not Legacy hardware. That's why this was forked.

**Comment by fallaha56** [comment]
Sure, I appreciate that, even if I feel the time of DX9 is passed -I'm more interested in DX12, never mind 11!

But the asynchronous shader problem, what about that? Or any kind of multithreaded acceleration?

Surely these merit consideration for inclusion?

Just some thoughts...

DF

**Comment by SukharevAndrey** [comment]
Dolphin developers dropped support of old API because of lacking support for it, they just don't want to do triple job. Ishiiruka's developer continues support for old APIs. So why can not they join forces?

**Comment by ADormant** [comment]
@SukharevAndrey Yeah and Ishiiruka has many other great features not only DX9. @Tinob

**Comment by ADormant** [comment]
@SukharevAndrey Yeah and Ishiiruka has many other great features not only DX9. @Tinob 
So I hope it'll be merged eventually to mainstream Dolphin.

**Comment by leoetlino** [comment]
Some features are merged back to upstream; however it takes a lot of work (or at least it used to) since the code style is not exactly the same, and sometimes they simply disagree with the way something has been implemented.
